aws_compliance.py

```diff
@@ -580,8 +622,9 @@ def get_cred_report():
     if "Fail" in status:
         return status
     response = IAM_CLIENT.get_credential_report()
+    responseString = str(response['Content'], 'utf-8').splitlines()
```

Comment:
Ah nice one - this is tricky python3 vs python2 stuff

Reply:
This was what I assumed as well. It's an easy enough fix though.